Instructions/03-agent-custom-functions.md

```diff
@@ -186,11 +169,11 @@ Now that you've created your project in AI Foundry, let's develop an app that im
 
     ```python
    # Connect to the Azure AI Foundry project
-   project_client = AIProjectClient.from_connection_string(
-        credential=DefaultAzureCredential
-            (exclude_environment_credential=True,
-             exclude_managed_identity_credential=True),
-        conn_str=PROJECT_CONNECTION_STRING
+   project_client = AIProjectClient(
```

Comment:
We might want to try to move this to agent_client? just update the includes to import AgentClient from azure.ai.agents